texting/migrations/0003_auto_20191126_1102.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.4 on 2019-11-26 11:02
+
+from django.db import migrations, models
+import project.util.phone_number
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('texting', '0002_phonenumberlookup'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='phonenumberlookup',
+            name='phone_number',
+            field=models.CharField(help_text='A U.S. phone number without parentheses or hyphens, e.g. "5551234567".', max_length=10, unique=True, validators=[project.util.phone_number.validate_phone_number]),
```

Comment:
This field formerly didn't include `validate_phone_number` as a validator, but it seems like it should.